Library/Homebrew/formula_creator.rb

```diff
@@ -100,8 +100,6 @@ class #{Formulary.class_s(name)} < Formula
           depends_on "cmake" => :build
```

Comment:
While you're here, can I suggest adding a line break before this `if` block so the `depends_on` statements stand alone?